README.md

```diff
@@ -1,157 +1,160 @@
-Nevertheless, we expect anyone interested in becoming a tripulante in Creditas to give it a try.
 
-In this challenge,cd we focus both on code design and design patterns in JavaScript. The goal is to assess your experience in writing **code that is easy to maintain, loosely coupled and highly cohesive**.
+In this challenge,we focus both on code design and design patterns in JavaScript.
 
-At Creditas we always work giving constructive feedback, therefore we will always dedicate the outmost attention to every solution submitted; contacting you back with the positive aspects of your submission and what could be improved. Just for that it is worth trying! :)
+At Creditas we always work giving constructive feedback, therefore we will always dedicate the outmost attention to every solution submitted, contacting you back with the positive aspects of your submission and what could be improved.
+
+Just for that it is worth trying! :)
 
 ## The challenge
 
-You have to improve the loan simulator implemented in this project.
+Cine Creditas is getting bigger and bigger and needs to create a self-service booking system.
 
-Both the interface as well as the styles are provided. The challenges include refactoring the code and implementing new functionalities (listed below).
+In order to make a booking, users must select a **film** and a **snack**.
 
-Feel free to componetize whatever you think should be componetized. Our only request is that you use only pure JavaScript, our beloved Vanilla. Do you accept the challenge?
+It is **important** to remark that our business model requires the user to select both elements ( a film and a snack )
 
-This is the simulator current state:
+Once the user has selected both items, they must confirm their selection and as a result a ticket will be displayed describing user's choice.
 
-<img style="display: block; margin: 0 auto;" src="./layout_en.png">
+Your mission will be to build a web application where the user can see both a list of films and a list of snacks, have the possibility to confirm their selection and display the corresponding generated ticket.
 
+Right now due to the traffic we are generating, we are averaging over 10 seconds to generate the ticket.
 
-### Expected functionalities
+As part of the challenge we need your help in order to reduce this delay as much as possible.
 
-The application must allow the user to choose the asset to be used as collateral:
-***"Auto"*** or ***"Home"*** (the default option is ***"Auto"*** ).
+## Resources
 
-It should also follow the calculation rules bellow:
+### Endpoints
 
-- Financial Transaction Tax (FTT): 6.38%
-- Interest Rate: 2.34%
-- Maximum loan amount (maximum amount the user can borrow) 80% of the collateral's value. This means that when you change the collateral value, **the maximum for the loan amount slider should be recalculated too**
+- GET https://frontend-challenge-vlc-api.public.dev.creditas.io/api/films
```

Comment:
This endpoint returns only 3 items. Maybe worth adding a bit more items (maybe +10? +20?) to also check the performance/experience on the mobile version. If there are bad performance practices in place, the experience will be bad in mobile, having only 3 items it's hard to spot that

Reply:
Also, what if we add random 2-3 broken images in the items in order to evaluate the ability to handle error cases?